package.json

```diff
@@ -223,6 +224,11 @@
                 "type": "boolean",
                 "description": "Whether the app should be rebuild before start.",
                 "default": true
+              },
+              "syncAllFiles": {
```

Comment:
Would you mind we rename this to "syncModules", syncModuleFiles, syncNodeModules... or something like that.

syncAllFiles is too generic and one  needs to read the description to know what that does.  the CLI flag is holding off the same I think.

@ivanbuhov @tzraikov